packages/generator-langium/templates/packages/language/langium-config.json

```diff
@@ -2,7 +2,7 @@
     "projectName": "<%= LanguageName %>",
     "languages": [{
         "id": "<%= language-id %>",
-        "grammar": "src/language/<%= language-id %>.langium",
+        "grammar": "src/<%= language-id %>.langium",
         "fileExtensions": <%= file-extension %>,
         "textMate": {
             "out": "syntaxes/<%= language-id %>.tmLanguage.json"
```

Comment:
Suggestion: The textmate grammar is generated into a folder in the `language` package. However, it is referenced the package.json of the VS Code extension as if it were located inside of the `extension` package. Unless the adopter changes the generated path or copies the file manually, they don't get syntax highlighting in VS Code.

Reply:
I added a build prepare step in both web and extension that copies the file over.